package.json

```diff
@@ -126,6 +126,12 @@
           "description": "Whether to enable references search for links in the editor. Reload required!",
           "type": "boolean"
         },
+        "memo.links.rules": {
+          "default": [],
+          "scope": "resource",
+          "markdownDescription": "Rules to create the files for short links. The file will be created in dir of **the first one in the list** that matches the filename.",
```

Comment:
```suggestion
          "description": "Rules specifying the location where to generate new files for short links. Applicable only if links format is set to short in the settings. In case if multiple rules matching the file path, the rule that comes first will be applied.",
```